pkg/spec3/operation.go

```diff
@@ -73,7 +73,7 @@ type OperationProps struct {
 	// Deprecated declares this operation to be deprecated
 	Deprecated bool `json:"deprecated,omitempty"`
 	// SecurityRequirement holds a declaration of which security mechanisms can be used for this operation
-	SecurityRequirement []*SecurityRequirement `json:"security,omitempty"`
+	SecurityRequirement []map[string][]string `json:"security,omitempty"`
```

Comment:
What's the reason behind that change? I guess you don't see any value in splitting vendor extensions from actual security requirements? I think it's unfortunate that one has to filter specifically the `x-...` when they are going through the list?

Reply:
Security Requirements do not support vendor extensions so having a separate struct seems unnecessary.